sdk/go/pulumi/types.go

```diff
@@ -654,6 +682,10 @@ func ToOutput(v interface{}) Output {
 // ToOutputWithContext returns an Output that will resolve when all Outputs contained in the given value have
 // resolved.
 func ToOutputWithContext(ctx context.Context, v interface{}) Output {
+	return toMaybeSecretOutputWithContext(ctx, v, false)
+}
+
+func toMaybeSecretOutputWithContext(ctx context.Context, v interface{}, forceSecret bool) Output {
```

Comment:
Nit: this is unnecessarily wordy. Let's let the parameter name speak for itself.

```suggestion
func toOutputWithContext(ctx context.Context, v interface{}, forceSecret bool) Output {
```